src/main/rx/srxl2.c

```diff
@@ -425,10 +425,8 @@ static uint8_t srxl2FrameStatus(rxRuntimeState_t *rxRuntimeState)
         result |= RX_FRAME_PROCESSING_REQUIRED;
     }
 
-    if (result == RX_FRAME_COMPLETE || result == (RX_FRAME_COMPLETE | RX_FRAME_PROCESSING_REQUIRED)) {
+    if (!(result & (RX_FRAME_FAILSAFE | RX_FRAME_DROPPED))) {
         lastRcFrameTimeUs = lastIdleTimestamp;
```

Comment:
@etracer65: Using the last idle time here sounds somewhat dodgy - but getting a better value for this might be hard, considering how the driver is structured. Have you tested this?

Reply:
Yes, I tested it with a Spektrum DX8e and SPM4651T receiver as noted in the original PR #9288. Before and after performance graphs are included.

The idle time logic was already part of the driver and a consequence of the way variable frame length handling was implemented. The protocol doesn't seem to have a good way to detect the start of a frame (no "start" byte) and they rely on the data stream going idle to indicate that the packet is complete.

Reply:
Using frame intervals to detect frame boundaries is actually a good way of doing things. And if this is what is done then this also means that the RX firmware will not suddenly be changed to introduce gaps between bytes outside of frame boundaries.